README.md

```diff
@@ -62,6 +62,18 @@ function NewComponent(props) {
 }
 ```
 
+### CDN
```

Comment:
😮 there doesn't need to be code changes? it's just ... here? I never knew!

Reply:
Neither did I, Edwin completely to the rescue. Now we know!